conf/application.conf

```diff
@@ -130,10 +130,6 @@ microservice {
         protocol = http
         host = localhost
         port = 9518
-        uris {
-          accessCode: "/ctc-guarantee-balance-eis-stub/guarantee/accessCode"
-          balance: "/ctc-guarantee-balance-eis-stub/guarantee/balance"
-        }
```

Comment:
These should remain, though we should probably just keep the prefixes, so `/ctc-guarantee-balance-eis-stub`, as that will still need to be configurable.